gostream/ffmpeg/avutil/avutil.go

```diff
@@ -73,6 +73,40 @@ func ptr(buf []byte) *C.uint8_t {
 	return (*C.uint8_t)(unsafe.Pointer(h.Data))
 }
 
+// SetFrameFromImgMacroAlign sets the frame from the given image.YCbCr adding
+// line padding to the image to ensure that the data is aligned to the given boundary.
+func (f *Frame) SetFrameFromImgMacroAlign(img *image.YCbCr, boundary int) {
```

Comment:
I know you've tested this with lots of different resolutions. Have you used different cameras and frame formats? I wonder if frame formats from different cameras have quirks that could break this.

Reply:
Tested with two different cams and YUYV/MJPEG pixel format source.